sklearn/utils/fixes.py

```diff
@@ -332,3 +334,51 @@ def _object_dtype_isnan(X):
     from collections import Iterable as _Iterable  # noqa
     from collections import Mapping as _Mapping  # noqa
     from collections import Sized as _Sized  # noqa
+
+
+def _joblib_parallel_args(**kwargs):
```

Comment:
Maybe just change it to `_joblib_parallel_kwargs` as you only set `kwargs` in it?

It would be clearer.